doc/connectivity/bluetooth/bluetooth-contrib.rst

```diff
@@ -0,0 +1,41 @@
+.. _bluetooth-contrib:
+
+Contributing to the Bluetooth subsystem
+#######################################
+
+Overview
+********
+
+This page describes how one would contribute to the Zephyr Bluetooth stack.
+It is a work in progress.
+
+The first step is familiarizing yourself with the :ref:`Stack architecture <bluetooth-arch>`.
+
+.. _bluetooth-coding-style:
+
+Coding guidelines
+*****************
+
+The following describes a few of the coding conventions we use in the Bluetooth
+subsystem. See also :ref:`Zephyr's Coding Guidelines <coding_guidelines` and the
+:ref:`Zephyr Coding Style <coding_style>`.
+
+The guidelines are not set in stone, and this is a living document. It will be
+updated if enough people agree on amending or changing the rules.
+
+Unless there is a convention listed here that disagrees, the general "rule" is
+to try to follow the style of the code surrounding the change.
+
+Tools
+=====
+
+Before submitting a pull request, it is a good idea to:
+
+- Run ``clang-format -i`` on your patch
```

Comment:
We could considerer referring to some common IDEs and extensions for those. Many use `clang-format` incorrectly  in the way that they end up formatting the entire file, rather than only changed lines. 

https://github.com/zephyrproject-rtos/zephyr/pull/75089 will also be a welcome addition here as that will generate comments in the review panel (but won't block PRs)

Reply:
I can add a link section at the bottom of the page.
editorconfig has a nice page listing all the plugins, but clang-format is all third-party.